utils/setup.cfg

```diff
@@ -3,3 +3,16 @@
 # 3. If at all possible, it is good practice to do this. If you cannot, you
 # will need to generate wheels for each Python version that you support.
 universal=1
+
+[nosetests]
+tests=../,../roles/openshift_master_facts/test/,test/
```

Comment:
@rhcarvalho I moved it, since we run the tests from the Makefile in utils and I was unable to get the tests running from the Makefile properly without having the setup.cfg in the same directory.

I'd be okay if we move the Makefile up to the root, this was just the path of least resistance to get the jenkins and travisci jobs to run all of the tests.  I'll fix the '-' issue' with a PR, but would need someone else to contribute the changes needed to move the Makefile to the root.